feat: bump hubble-sdk  v0.22.2, show local location of Executors in Hub (#5282)
jina-ai/jina: docs/fundamentals/executor/hub/use-hub-executor.md

jina hub pull jinahub://<UUID>[:<SECRET>][/<TAG>]
```

### List locations of local Executors

```bash
jina hub list
```

<script id="asciicast-z81wi9gwVm7gYjfl5ocBD1RH3" src="https://asciinema.org/a/z81wi9gwVm7gYjfl5ocBD1RH3.js" async></script>

```{tip}

> `jina hub list` command only lists all the local Executors with source code format. 

Reviewer: ```suggestion
To list all the Executors that are in source-code format (i.e. pulled via `jinahub://`), use the command `jina hub list`
```

Author: I will convert  ``jina hub list` command only lists all the local Executors with source code format.`   to  T`o list all the Executors that are in source-code format (i.e. pulled via `jinahub://`), use the command `jina hub list``

> If you want to list all the docker format Executors, you can explictly use `docker images`. 

Reviewer: ```suggestion
To list all the Executors that are in Docker format, use the command `docker images`. 
```

Be sure to check spelling in future!

jina hub pull jinahub://<UUID>[:<SECRET>][/<TAG>]
```

### List locations of local Executors

```bash
jina hub list
```

<script id="asciicast-z81wi9gwVm7gYjfl5ocBD1RH3" src="https://asciinema.org/a/z81wi9gwVm7gYjfl5ocBD1RH3.js" async></script>

```{tip}

> To list all the Executors that are in source-code format (i.e. pulled via `jinahub://`), use the command `jina hub list`

Reviewer: ```suggestion
To list all the Executors that are in source-code format (i.e. pulled via `jinahub://`), use the command `jina hub list`.
```

The source code of the Executor will be stored at `~/.jina/hub-packages`.

> ### list the local location of executors

Reviewer: For headings, ensure:

- Top level headings (`# foo`) - [Title Case](https://titlecaseconverter.com/#TitleCapitalizationRules)
- Lower level headings (`## bar`, `### baz`) - [Sentence case](https://titlecaseconverter.com/#WhatIsSentenceCase)

Links above go to a tool that will auto-format your text accordingly.

We use AP style

Author: Thanks for the suggestion, I will fix it.

> ### list the local location of executors

Reviewer: ```suggestion
### List locations of local Executors
```

Author: Thanks, i will convert  `list the local location of executors`  to` List locations of local Executors`

```bash

> jina hub list

Reviewer 2: can u please add an output example?

Author: We'll record a command line video about it

```

> Show the local location of Jina executors.

Reviewer: This is just rewording of the title. We need something better here, or just remove it altogether

Author: I will remove it and add the command line video.
